### PR TITLE
Monkey patch Tap.cli to accept synonymous args

### PR DESCRIPTION
**Pull request by vorpal56**
We pin the version, so I'm not too concerned about this. However if we ever do bump up `singer_sdk`, it might be necessary to update this.

### CODE REVIEW

File: tap_aircall/_tap.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        @common_options.PLUGIN_ABOUT
+        @common_options.PLUGIN_ABOUT_FORMAT
+        @common_options.PLUGIN_CONFIG
+        @click.option(
+            "-c",
+            multiple=True,
+            help="Configuration file location or 'ENV' to use environment variables.",
+            type=click.STRING,
+            default=(),
+        )
```
Comment (vorpal56) [inline tap_aircall/_tap.py]: The addition is here

File: tap_aircall/_tap.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+            help="Execute the Singer tap.",
+            context_settings={"help_option_names": ["--help"]},
+        )
+        def cli(
+            version: bool = False,
+            about: bool = False,
+            discover: bool = False,
+            test: CliTestOptionValue = CliTestOptionValue.Disabled,
+            config: Tuple[str, ...] = (),
+            c: Tuple[str, ...] = (),
```
Comment (vorpal56) [inline tap_aircall/_tap.py:86]: Here

File: tap_aircall/_tap.py [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+                state: Use a bookmarks file for incremental replication.
+
+            Raises:
+                FileNotFoundError: If the config file does not exist.
+            """
+            if version:
+                cls.print_version()
+                return
+
+            config_to_use = config or c
```
Comment (vorpal56) [inline tap_aircall/_tap.py:114]: And here

**Comment by cscott20** [comment]
Hmmm, still getting the same error on this branch:
image

**Comment by vorpal56** [comment]
@cscott20 did you reinstall using `pip install .`?

**Comment by cscott20** [comment]
Oop, thank you, that fixed the issue.

**Comment by cscott20** [comment]
I think we are going to need to make the same change so that "properties" is also an accepted argument.
Or as an alternative, ignore any unknown additional flags

**Review by cscott20 — APPROVED** [review]

**Pull request merged**